$NetBSD: patch-configure,v 1.4 2020/02/20 20:39:07 he Exp $

Pretend expat.h is found: it is guaranteed by PkgSrc, but on Darwin it might
be buried inside an SDK; we don't want the SDK path being exposed in CFLAGS.

--- configure.orig	2017-07-09 07:41:42.000000000 +0000
+++ configure
@@ -19489,7 +19489,7 @@ fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for libexpat" >&5
 $as_echo_n "checking for libexpat... " >&6; }
-found_libexpat="no"
+found_libexpat="yes"
 for dir in $withval ; do
             if test -f "$dir/include/expat.h"; then
 		found_libexpat="yes"
